frontend/src/components/ResetPasswordCard/ResetPasswordCard.js

```diff
@@ -0,0 +1,187 @@
+import React, { useState } from 'react';
+import {
+  Button,
+  CardContent,
+  Grid,
+  IconButton,
+  InputAdornment,
+  Paper,
+  TextField,
+  Typography,
+} from '@material-ui/core';
+import { PASSWORD_REGEX } from '../../constants';
+import {
+  forgotPasswordDrawing,
+  visibilityIcon,
+  visibilityOffIcon,
+} from '../../assets';
+
+const ResetPasswordCard = ({ onSend }) => {
+  const [showPasswordError, setShowPasswordError] = useState(false);
```

Comment:
Move this state to bottom of states. Move `showConfirmPasswordError` as well. This way we will keep the logical grouping of states. First come the values, then errors, etc.
I know it's another nitpicky suggestion. Feel free to object.

Reply:
It's fine, it's a quick change